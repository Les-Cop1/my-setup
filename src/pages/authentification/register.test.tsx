import React from 'react'

import { Register } from '@pages'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import { MemoryRouter } from 'react-router-dom'

describe('Register', () => {
  test('Snapshot', () => {
    const { container } = render(<Register />, { wrapper: MemoryRouter })
    expect(container).toMatchSnapshot()
  })
})
