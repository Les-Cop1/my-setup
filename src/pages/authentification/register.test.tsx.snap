// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register Snapshot 1`] = `
<div>
  <div
    class="flex min-h-full flex-col justify-center py-12 sm:px-6 lg:px-8"
  >
    <div
      class="sm:mx-auto sm:w-full sm:max-w-md"
    >
      <div
        class="flex justify-center flex-shrink-0 text-6xl mx-auto h-12 w-auto"
      >
        <span
          class="sr-only"
        >
          mySetup
        </span>
        üè†
      </div>
      <h2
        class="mt-6 text-center text-3xl font-bold tracking-tight text-slate-900"
      >
        Create an account
      </h2>
    </div>
    <div
      class="mt-8 sm:mx-auto sm:w-full sm:max-w-md"
    >
      <div
        class="bg-white py-8 px-4 shadow sm:rounded-lg sm:px-10"
      >
        <form
          class="space-y-6"
        >
          <div>
            <label
              class="block text-sm font-medium text-slate-700"
              for="username"
            >
              Username
            </label>
            <div
              class="relative mt-1 rounded-md shadow-sm"
            >
              <input
                autocomplete="username"
                class="block w-full rounded-md shadow-sm placeholder-slate-400 sm:text-sm border-slate-300 focus:border-teal-500 focus:ring-teal-500"
                id="username"
                name="username"
                required=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div>
            <label
              class="block text-sm font-medium text-slate-700"
              for="password"
            >
              Password
            </label>
            <div
              class="relative mt-1 rounded-md shadow-sm"
            >
              <input
                autocomplete="new-password"
                class="block w-full rounded-md shadow-sm placeholder-slate-400 sm:text-sm border-slate-300 focus:border-teal-500 focus:ring-teal-500"
                id="password"
                name="password"
                required=""
                type="password"
                value=""
              />
            </div>
          </div>
          <div>
            <label
              class="block text-sm font-medium text-slate-700"
              for="confirmation"
            >
              Confirmation
            </label>
            <div
              class="relative mt-1 rounded-md shadow-sm"
            >
              <input
                autocomplete="new-password"
                class="block w-full rounded-md shadow-sm placeholder-slate-400 sm:text-sm border-slate-300 focus:border-teal-500 focus:ring-teal-500"
                id="confirmation"
                name="confirmation"
                required=""
                type="password"
                value=""
              />
            </div>
          </div>
          <div>
            <button
              class="flex w-full justify-center rounded-md border border-transparent bg-teal-600 py-2 px-4 text-sm font-medium text-white shadow-sm hover:bg-teal-700 focus:outline-none focus:ring-2 focus:ring-teal-500 focus:ring-offset-2"
              type="submit"
            >
              Sign up
            </button>
          </div>
        </form>
        <div
          class="mt-6"
        >
          <div
            class="relative"
          >
            <div
              class="absolute inset-0 flex items-center"
            >
              <div
                class="w-full border-t border-slate-300"
              />
            </div>
            <div
              class="relative flex justify-center text-sm"
            >
              <span
                class="bg-white px-2 text-slate-500"
              >
                <a
                  class="font-medium text-teal-600 hover:text-teal-500"
                  href="/login"
                >
                  or sign in
                </a>
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
